autogluon/training/docker/0.4.0/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,131 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+ARG PYTHON_VERSION=3.8.12
+ARG PYTHON_VER=3.8
+ARG FROM_IMAGE=rapidsai/rapidsai-core
+
+FROM ${FROM_IMAGE}:cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER}
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG CUDA_VER
+
+ARG OPENSSL_VERSION=1.1.1l
```

Comment:
Please update this to the latest openssl version (I think `1.1.1n`).

Reply:
done